infra/feast-operator/Makefile

```diff
@@ -48,11 +48,11 @@ endif
 
 # Set the Operator SDK version to use. By default, what is installed on the system is used.
 # This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
-OPERATOR_SDK_VERSION ?= v1.37.0
+OPERATOR_SDK_VERSION ?= v1.38.0
```

Comment:
WDYT to use release v1.39.0-> https://github.com/operator-framework/operator-sdk/tree/v1.39.0